cmd/kubeadm/app/master/manifests.go

```diff
@@ -296,6 +297,24 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 		"--allow-privileged",
 	)
 
+	if cfg.AuthorizationMode != "" {
+		switch cfg.AuthorizationMode {
+		case authorizer.ModeABAC:
+			command = append(command, "--authorization-mode="+cfg.AuthorizationMode,
+				"--authorization-policy-file="+kubeadmapi.GlobalEnvParams.KubernetesDir+"/policy_file.json",
+			)
+		case authorizer.ModeRBAC:
+			command = append(command, "--authorization-mode="+cfg.AuthorizationMode,
+				"--runtime-config=rbac.authorization.k8s.io/v1alpha1",
```

Comment:
Has rbac made v1beta1 at HEAD yet?
In the next v1.6.0-alpha.x I suppose we can switch to beta directly? 
I don't think we want to support rbac in alpha, do we?